ovirt-engine-hyper-upgrade.py

```diff
@@ -74,9 +66,10 @@ def get_enabled_repos(self):
         '''
         self.repos = []
 
-        output = execute_cmd(
-             self.SUBSCRIPTION_MANAGER_GET_ENABLED_REPOS
-        ).splitlines()
+        rhsm_cmd = list(self.SUBSCRIPTION_MANAGER)
```

Comment:
IIUC you use 'list' because you want a copy. Some other options:

1. Use plain '+' - execute_cmd(self.SUBSCRIPTION_MANAGER + ['repos', '--list-enabled'])

2. Or a more efficient variation 'itertools.chain' - probably an overkill here - https://stackoverflow.com/questions/1720421/how-to-concatenate-two-lists-in-python

3. If you do want to copy, I know it's common to use a slice, self.SUBSCRIPTION_MANAGER[:] . Not sure why it's common, but someone tested and said it's the fastest: https://stackoverflow.com/questions/2612802/how-to-clone-or-copy-a-list/2612990#2612990